ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20.yaml

```diff
@@ -539,16 +539,28 @@ tests:
     workflow: openshift-e2e-vsphere
   timeout: 6h0m0s
 - always_run: false
-  as: e2e-aws-ovn-virt
+  as: e2e-metal-ipi-ovn-virt-dualstack
+  cluster: build05
   optional: true
   steps:
-    cluster_profile: aws-3
+    cluster_profile: equinix-ocp-metal
     env:
       CNV_PRERELEASE_LATEST_CHANNEL: "true"
-      COMPUTE_NODE_REPLICAS: "2"
```

Comment:
how is this controlled in this new cluster deployment ? are 2 nodes the default ?